### PR TITLE
Fix typo in util.py

### PR DESCRIPTION
**Pull request by eltociear**
### Description

identifer -> identifier

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

### CODE REVIEW

**Comment by eltociear** [comment]
I signed CLA.

**Comment by leahwicz** [comment]
@cla-bot check

**Comment by jtcohen6** [comment]
@eltociear Thanks for the PR! We wouldn't normally require a changelog entry for changes to non-functional code, but we also want to give you credit for the contribution :) think you could [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-a-changelog-entry) and push it up?

File: .changes/unreleased/Fixes-20221117-220320.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+kind: Fixes
+body: Fix typo in util.py
+time: 2022-11-17T22:03:20.4836855+09:00
+custom:
+  Author: eltociear
+  Issue: "6037"
```
Comment (emmyoop) [inline .changes/unreleased/Fixes-20221117-220320.yaml:6]: We use a specific issue for changes that do not have associated issues.  It's the result of how our changelog is currently implemented.
```suggestion
  Issue: "4904"
```

**Review by emmyoop — CHANGES_REQUESTED** [review]
Update the changelog and this should be good to go!  Thanks for the contribution.

**Review by emmyoop — APPROVED** [review]

**Pull request merged**